src/Common/Common.Api/Datasets/DatasetsClient.cs

```diff
@@ -16,6 +16,15 @@ public DatasetsClient(IPowerBIClient client) : base(client)
         {
         }
 
+        public object AddDataset(Dataset dataset, Guid? workspaceId = default)
+        {
+            var result = workspaceId.HasValue && workspaceId.Value != default ?
```

Comment:
The formatting on this assignment is a bit off.

Reply:
Thanks. I fixed it.